Sources/Publish/Internal/Collection+ContainsPrefix.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Collection+ContainsPrefix.swift
+//  
+//
+//  Created by Ben Syverson on 2020/01/28.
+//
+
+import Foundation
+
+internal extension Collection where Element: StringWrapper {
+    func containsPrefixFor(_ string: String) -> Bool {
+        reduce(false){ $0 || string.hasPrefix($1.string) }
```

Comment:
This will needlessly iterate through the collection every time, even if a match was found, making us turn an `O(1)` operation (checking if a set contains a member) into an operation that *always* is `O(N)` (not only in the worst case). I would suggest using the `contains(where:)` API instead:

```swift
contains(where: { $0.string.hasPrefix(string) })
```